### ISSUE TITLE
A11y: SUI checkboxes are not accessible

### ISSUE THREAD

**Issue by jennur** [inveniosoftware/react-searchkit#251]
There is a a11y bug with the SUI checkboxes, discovered in the `BucketAggregationValues` where the SUI `Checkbox` is used. We need to find a solution to this to be compliant with WCAG.

The issue seems to be how SUI renders the checkbox input: The input is hidden and has a label element next to it, but they are not associated. Screenshot from WAVE check:

<img width="805" alt="Screenshot 2023-08-17 at 16 31 14" src="https://github.com/inveniosoftware/react-searchkit/assets/21052053/16fcc204-ab05-4f69-bd94-1c962e3e96e0">

**Comment by jennur** [comment]
Sorry for the noise (too fast), after a double checking, this was not related to the BucketAggregationValues after all.
